app/settings/addons/styles.scss

```diff
@@ -0,0 +1,9 @@
+.PanelBody {
+    .PanelSection {
+        border-bottom: 1px solid #eee;
+    }
+
+    .PanelSection:last-child {
+        border: 0;
+    }
+}
```

Comment:
IMHO these styles belong to the list and not the AddonSection. When you move to `paginated-list/all`, the `AddonSection` will be wrapped in a `<li>` that you can apply these styles to.